### ISSUE TITLE
Request: Animation When Clearing Notifications

### ISSUE THREAD

**Issue by YSNBx** [ioscreatix/Intelix#98]
Love this tweak but it would be nice if we could get an animation when clearing notifications. They just abruptly disappear when 'Clear All' is pressed.

**Comment by JeffResc** [comment]
I totally agree with this and wanted to also request this feature. When no animation is played it looks to me like it glitched because it just disappears and the notifications below just also appear higher which I believe would look better with some sort of fade/float animation that stock iOS included

**Comment by alejara93** [comment]
Agree! It seems to disappear
